kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/language/scene/language/GodotSceneLanguage.kt

```diff
@@ -0,0 +1,14 @@
+package godot.intellij.plugin.language.scene.language
+
+import com.intellij.lang.Language
+
+class GodotSceneLanguage private constructor() : Language("godotScene") {
+    override fun getDisplayName(): String = NAME
+    override fun isCaseSensitive(): Boolean = true
+
+    companion object {
```

Comment:
Is that for Java compatibility issues ?

Reply:
AFAIK yes.
`GodotSceneFileType` for example is referenced by such a field in the `plugin.xml`:
```xml
<fileType name="GodotSceneFile"
            fieldName="INSTANCE"
            language="godotScene"
            extensions="tscn"
            implementationClass="godot.intellij.plugin.language.scene.filetype.GodotSceneFileType"/>
```

For the `GodotSceneLanguage` i assumed it must be the same as all implementations i checked (all tough all written in plain java) have such an `INSTANCE` field.